corehq/apps/accounting/tasks.py

```diff
@@ -227,6 +235,19 @@ def send_subscription_reminder_emails(num_days, exclude_trials=True):
             subscription.send_ending_reminder_email()
 
 
+def send_subscription_reminder_emails_dimagi_contact(num_days):
+    today = datetime.date.today()
+    date_in_n_days = today + datetime.timedelta(days=num_days)
+    ending_subscriptions = (Subscription.objects
+                            .filter(is_active=True)
+                            .filter(date_end=date_in_n_days)
```

Comment:
Is it true that the alert on a subscription would not be sent if the task did not run for whatever reason (Celery was stopped due to deploy, etc.) on the day that it's end date matches `date_in_n_days`? Does it matter if that happens occasionally?